.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+    "presets": [
+      "@babel/preset-env"
+    ],
+    "plugins": ["@babel/plugin-transform-runtime"]
```

Comment:
As mentioned, needed this to for babel to compile the loggers.
https://stackoverflow.com/questions/61755999/uncaught-referenceerror-regeneratorruntime-is-not-defined-in-react